### ISSUE TITLE
No demo

### ISSUE THREAD

**Issue by oleteacher** [ckeditor/ckeditor5-demos#18]
Your demo link does not work:

http://ckeditor.com/ckeditor-5/demo/html-source-editing/

**Comment by mlewand** [comment]
Thanks for reporting, indeed looks like this link was changed. 

Am I right assuming that you found this link here: https://github.com/ckeditor/ckeditor5-demos/blob/master/source-code-editing/README.md - there's a link in the "See this demo live at **ckeditor.com** or read"? If not can you tell where you have found it?

Naturally, the proper link is:

[https://ckeditor.com/ckeditor-5/demo/html-support/](https://ckeditor.com/ckeditor-5/demo/html-support/)

Let's do the following:

1.  Update the link in the [README.md](https://github.com/ckeditor/ckeditor5-demos/blob/master/source-code-editing/README.md)
2.  Add a redirect for the old link in our website, so that it redirects old links just in case.

**Comment by oleteacher** [comment]
Yes, it was on https://github.com/ckeditor/ckeditor5-demos/blob/master/source-code-editing/README.md

My bad, I should have noted that in report:)

**Comment by mlewand** [comment]
Fixed, thanks!
